tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -207,7 +208,9 @@ def run_hyperopt_executor(
 def test_hyperopt_executor(scenario, csv_filename, tmpdir, ray_cluster_4cpu):
     search_alg = scenario["search_alg"]
     executor = scenario["executor"]
-    run_hyperopt_executor(search_alg, executor, csv_filename, tmpdir)
+    # When using the hb_bohb scheduler, num_epochs must equal max_t (which is 81 by default)
```

Comment:
How did you come to discover this? Is this worth adding as an auxiliary validation?

Reply:
I didn't know about this either, but found out that we've added this logic here in our ModelConfig object creation logic: https://github.com/ludwig-ai/ludwig/blob/master/ludwig/schema/model_config.py#L532